packages/@react-spectrum/s2/src/Badge.tsx

```diff
@@ -165,12 +165,12 @@ const badge = style<BadgeStyleProps>({
     fillStyle: {
       outline: {
         variant: {
-          accent: 'accent-800',
-          informative: 'informative-800',
-          neutral: 'gray-700',
-          positive: 'positive-700',
-          notice: 'notice-700',
-          negative: 'negative-800'
+          accent: lightDark('blue-800', 'blue-900'), // accent-visual-color
```

Comment:
is there are reason why we aren't using 'accent-800'/'accent-900' instead? that would more closely match how they describe it in the token figma file

Reply:
good call! I'm not sure why I dove down an extra layer when getting the colors haha